website.qmd

```diff
@@ -0,0 +1,9 @@
+### Package website modes
+
+Epiverse-TRACE packages use the [`{pkgdown}` package](https://pkgdown.r-lib.org/) to generate websites. `{pkgdown}` provides [development modes](https://pkgdown.r-lib.org/reference/build_site.html#development-mode) (in the [`_pkgdown.yml` file](https://pkgdown.r-lib.org/articles/customise.html)) to generate different sites for the development and released versions of your package.
+
+When the package is not yet on CRAN, we use the `unreleased` development mode. In this case, we use a visual element to highlight the package development version in red. An example is the [website](https://epiverse-trace.github.io/superspreading/) for `{superspreading}` and its associated `_pkgdown` [configuration](https://github.com/epiverse-trace/superspreading/blob/main/_pkgdown.yml#L12-L13).
```

Comment:
Could expand the hyperlink to include "website for `{superspreading}`, but it's personal preference.

Reply:
I've done that in [677d2aa](https://github.com/epiverse-trace/blueprints/pull/125/commits/677d2aadad9701379c257f8d9437d30c29456dae).